### ISSUE TITLE
Vue-Storybook props values column and array control

### ISSUE THREAD

**Issue by saurabh-sali-hulkapps** [storybookjs/storybook#15717]
@shilman 

How to add a new column to the docs table in Vue-Storybook?
Like,
![image](https://user-images.githubusercontent.com/82803146/127601994-5b26e79e-f979-42eb-a266-bac809214e74.png)

And how to get default as an array instead of object in controls?
![image (1)](https://user-images.githubusercontent.com/82803146/127602072-e8ad7980-5f1c-4802-af53-45b2c3e9d1d1.png)

**Comment by shilman** [comment]
@saurabh-sali-hulkapps we don't allow you to add new columns to the ArgsTable

**Comment by saurabh-sali-hulkapps** [comment]
I need another column to display, I want to show the **props** values. And what about the array control?

**Comment by shilman** [comment]
@ghengeveld is there something we need to do about array values in the object editor?

**Comment by ghengeveld** [comment]
Yeah we should set a better default that takes into account the data type.

**Comment by valentinpalkovic** [comment]
Thanks for the question. The `ArgsTable` doc block was removed in v8 (replaced by `ArgTypes` and `Controls`, which don't expose custom columns by design), and Vue 2 support was dropped in v8 — so the Vue-2 argType inference quirks aren't reproducible in Storybook 10. Closing as obsolete.
